src/index.ts

```diff
@@ -107,6 +107,7 @@ export async function main(config: Output<typeof Config>) {
     let commit_scope = await p.select({
       message: "Select a commit scope",
       initialValue: config.commit_scope.initial_value,
+      maxItems: config.commit_scope.max_items,
```

Comment:
On most PRs I would say this is exactly what we want to do (thanks for taking the time to understand the repo!). - I think in this case we could just bump the version and put `maxItems: Infinity` (Or just a relatively high number lol) here and on `commit_type`. 

IIRC (maybe it has changed) clack doesn't support filter / search on select. As an example, if we had 10 scopes but a max_items of 5, we don't have a way to filter where it would show any of the 5 hidden items.

Reply:
I added `max_items` for commit_type. Yes, we have the ability to specify what will be displayed, but if we specify `initialValue` then p.select will correctly move through the list:

<img width="582" alt="image" src="https://github.com/Everduin94/better-commits/assets/682584/f1d25e23-a6fa-4b13-bf9d-ec2207560f28">

Reply:
👀 I misunderstood how max-items works. - I'll merge :), thanks again!